PrecompiledWeb/gsa-simulator/Web.config

```diff
@@ -1,18 +1,20 @@
 <?xml version="1.0"?>
 <configuration>
-	<system.web>
-		<!--  DYNAMIC DEBUG COMPILATION
+  <system.codedom>
+    <compilers>
+      <!--  DYNAMIC DEBUG COMPILATION

```

Comment:
This comment may no longer be accurate, since there is no debug="true" option.  Maybe change to match compilerOptions, if needed, or remove it if no longer applicable.